### ISSUE TITLE
Would you supply the codes of part about SOMs (self-organizing maps)?

### ISSUE THREAD

**Issue by tbwxmu** [insilicomedicine/GENTRL#3]
Hi all, 
I read the paper and its supplement document. I found Table 2 in Supplementary missing the Trend SOM as described in the 4th paragraph of the introduction.  I want to reproduce your calculation results as you did in the paper, but I find you did not supply the analysis codes and the links of the raw data sets you used.
I hope you add the missing parts and make it possible for others to repeat your calculation results.

look forward to your reply

**Comment by danpol** [comment]
Regarding Table 2, we used all three SOMs as reward functions but applied only general and specific kinase SOMs for final prioritization as described in the paper. In this repository, we provide the central part of the pipeline used for DDR1 — GENTRL itself with an example application on a simple reward function. 

Regarding the datasets, we provided detailed information (including sources and number of structures) in the Supplementary Table 1 and the main text/methods. The whole pipeline was replicated by multiple groups in the pharmaceutical industry last week, so collecting the data should not be a problem.

**Comment by Bibyutatsu** [comment]
Hi, can you please tell me how to reproduce the SOMs and what is the significance of the colours in the graph?
Also what is the training data used for these SOMs?

**Comment by tbwxmu** [comment]
Hi Mitra,
Unluckily, they did not supply the raw data as you see they ask other
researchers to collect the data from the web with their described. For now,
with no prepared datasets and the pipeline codes, as they used, I think
only God can reproduce their results except themselves.
Last week, I learned the SOM by myself. It is a very old and simple network
for clustering. I also implanted it with PyTorch, if you are interested.
here:https://github.com/tbwxmu/MolD_pipeline/tree/master/PiPline


Bibhash Chandra Mitra <notifications@github.com> 于2019年10月3日周四 上午6:08写道：

**Comment by Bibyutatsu** [comment]
Hi tbwxmu,
Thank you very much. I am also trying to reproduce the results. Its just in the reward function that I am unable to construct the SOM as I don't know what features they used to train the SOMs

**Comment by tbwxmu** [comment]
Hi Bibutatsu,
I think they used the molecule fingerprint as the feature to train SOM. If you read my code, you should easily get the similar SOM maps as they plot in supplementary.

**Comment by Bibyutatsu** [comment]
Hi tbwxmu,
Can you share the plot from your code? 
Because I am able to see only this plot which looks scattered to me.

![download (2)](https://user-images.githubusercontent.com/11424274/66738118-61315a80-ee8b-11e9-8042-573471dde984.png)

**Comment by tbwxmu** [comment]
![test_heatmapF0](https://user-images.githubusercontent.com/53101398/66783302-40720080-ee9d-11e9-9a91-d3251c96597b.png)

**Comment by Bibyutatsu** [comment]
Hi tbwxmu,
Thanks for the plot. Yes it is very similar to the one in the supplementary.
